include/dgl/array.h

```diff
@@ -56,6 +56,18 @@ inline SparseFormat ParseSparseFormat(const std::string& name) {
     return SparseFormat::kAny;
 }
 
+// Create string from sparse format.
+inline std::string ToStringSparseFormat(SparseFormat sparse_format) {
+  if (sparse_format == SparseFormat::kCOO)
+    return std::string("coo");
+  else if (sparse_format == SparseFormat::kCSR)
+    return std::string("csr");
+  else if (sparse_format == SparseFormat::kCSC)
+    return std::string("csc");
+  else
+    return std::string("any");
+}
```

Comment:
Since `SparseFormat` is a new style enum class, you could define type conversion operator for it:
```c++
operator std::string() const;
```

Reply:
I'm not sure if it's true, https://stackoverflow.com/questions/5093460/how-to-convert-an-enum-type-variable-to-a-string it's said enum is not a class in c++.

Reply:
SparseFormat is a scoped enum class instead of an old style enum. Scoped enum class is strongly-typed, so I could it should allow operator overloading.